.eslintrc.js

```diff
@@ -552,29 +552,6 @@ module.exports = {
       },
     },
 
-    /**
```

Comment:
:information_source: Given that we've moved the Graph plugin and CI still runs fine since then, I assume those rules (since they just loosen rules not making them more strict) are no longer required :tada:

Reply:
Yes, @maryia-lapata fixed those when migrating - thanks @maryia-lapata !